src/TTextEdit.cpp

```diff
@@ -54,6 +54,9 @@ TTextEdit::TTextEdit(TConsole* pC, QWidget* pW, TBuffer* pB, Host* pH, bool isLo
 , mIsCommandPopup(false)
 , mIsTailMode(true)
 , mShowTimeStamps(false)
+#if !defined(QT_NO_DEBUG)
```

Comment:
Double negative here? Thought you didn't like them!

Reply:
I do not but it is not down to me as it is the QT symbol - that **is** `QT_NO_DEBUG` which will be defined for release builds without debug info - so the inverse is true for debug builds.

Reply:
We already use `QT_DEBUG` which is more straightforward to understand, why can't it work here?

Reply:
I wasn't aware of that one - I do not know when it was introduced and the on-line research suggested that the other one predated it...

Reply:
... but yeah it does look to be sensible to use it.